apps/service/handlers.go

```diff
@@ -106,7 +106,16 @@ func getTallyHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	tally, err := model.GetTally(db, campaign)
+	// This is allowed to be empty
+	county := queryParams.Get("county")
+	zipCodes := countyZips[county]
```

Comment:
i feel like ideally this should not be case sensitive? i'd prob make everything lowercase in the data file. should be super quick to transform it.